src/move_basic.cpp

```diff
@@ -173,6 +175,9 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(30.0)),
     // how long to wait for an obstacle to disappear
     nh.param<double>("obstacle_wait_limit", obstacleWaitLimit, 10.0);
 
+    // Reverse distances for which rotation won't be performed
+    nh.param<double>("straight_back_threshold", straightBackThreshold, 0.5);
```

Comment:
'straight_back' is a misleading name, it should be backup or reverse instead.

Reply:
Changed